lib/examples/axum/.env.example

```diff
@@ -0,0 +1,2 @@
+# SurrealDB connection endpoint
+SURREALDB_ENDPOINT=ws://your_surrealdb_host:port
```

Comment:
I can see how it's useful to include an example on how one can use `dotenv` to setup the environment variable like this but I don't think it's necessary. I think we should leave this as an exercise to the user and avoid adding one more dependency that we would need to keep track of. Besides, there are many ways one can setup the environment variable. By including `dotenv` like this, some users might actually think they are required to use it.